### ISSUE TITLE
Letter Head footer changes web URL to link

### ISSUE THREAD

**Issue by bedosk** [frappe/erpnext#9150]
As discussed in https://discuss.erpnext.com/t/letter-head-footer-changes-web-url-to-link/17198
When using code view:
![image](https://cloud.githubusercontent.com/assets/21075150/26795829/5dd1eeb4-4a27-11e7-8869-7e75495316cb.png)

the text will get automatically changed after saving to:

print preview:
![image](https://cloud.githubusercontent.com/assets/21075150/26795628/a7f5e898-4a26-11e7-9740-1a944e073651.png)
pdf: 
![image](https://cloud.githubusercontent.com/assets/21075150/26795666/c7ffb470-4a26-11e7-84fd-9b8f5a2da132.png)

There is one strange thing - when using code view:
1) code: `www.test.com `- will stay and output correctly
2) code `<b>www.test.com</b>` - will change after saving to `<b><a href="http://www.test.com">www.test.com</a></b>` and subsequently pdf will create url in brackets. I can reproduce this every time.

**Comment by rohitwaghchaure** [comment]
Hi @bedosk 

In frappe, standard python library pdfkit has used to transfer the html to pdf and pdfkit change the 
```
<a href="https://www.test.com">Test Data</a> 
```
to
```
Test Data (https://www.test.com)
```
But because of this, you can able to click and open the url in the tab from the PDF file

**Comment by bedosk** [comment]
hi @rohitwaghchaure,

I'm sorry but I think you are mistaken. 
I'm not disputing that <a href="http://www.test.com">Test Data</a> will be changed to Test Data (http://www.test.com)

What I'm saying is that a simple <br>www.test.com</br> is somehow automatically changed to <a href="http://www.test.com">www.test,com</a>

This is a big difference. First of all I don't want system to change my code. Second of all, it seems pdfkit only does that for <a href></a>. Not for any other possible link, like email@test.com

Bottom line is, user should be able to decide what he prefers. I prefer that my html will not be altered. If I choose to use <a href> then I will accept link in (), but in case I just type www.test.com I'd like to see just www.test.com.
